### PR TITLE
[ON HOLD] Safari fixes

### PR DESCRIPTION
**Pull request by karimkawambwa**
## Description

- [x] Safari navigation.

Screen Shot 2019-06-24 at 8 49 22 AM
Screen Shot 2019-06-24 at 8 49 00 AM

- [x] Safari Hurumap data scroll.

Fixes # (issue)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: takwimu/templates/profile/profile_detail_takwimu.html [hunk 1]
```diff
@@ -4,7 +4,14 @@
 {% block profile_header_tail %}
 {% endblock %}
 
-
+{% block head_css_extra %}
```
Comment (kilemensi) [inline takwimu/templates/profile/profile_detail_takwimu.html]: QQ: Why are these here and not in `takwimu.scss` @karimkawambwa?

Comment (karimkawambwa) [reply takwimu/templates/profile/profile_detail_takwimu.html]: At the moment fix, I guess i can move it there.

Comment (karimkawambwa) [reply takwimu/templates/profile/profile_detail_takwimu.html]: Lets see if @nerdlet  notices that the fix works. Hopefully she has the time.

**Comment by kilemensi** [comment]
Please review this PR @nerdlet

**Review by nerdlet — COMMENTED** [review]
@karimkawambwa @kilemensi Just checked it on safari. LGTG
image

**Comment by karimkawambwa** [comment]
@nerdlet did you verify scrolls work `Safari Hurumap data scroll.`?

**Comment by kilemensi** [comment]
Unfortunately @karimkawambwa this PR completely does the opposite in FF:

Screenshot_2019-06-25 Takwimu profile Tanzania

Screenshot_2019-06-25 Takwimu profile Tanzania(1)

We need to further test it, in a cross-platform way.

**Comment by kilemensi** [comment]
Archiving repo
